tensorboard/data/server/pip_package/BUILD

```diff
@@ -0,0 +1,39 @@
+package(default_visibility = ["//tensorboard:internal"])
```

Comment:
Not a big harm but does this need to be visible to other packages given that it has no libraries?